File: jfingerprintmanager/src/main/java/com/jesusm/jfingerprintmanager/encryption/Base64Encoder.java
package com.jesusm.jfingerprintmanager.encryption;

import android.util.Base64;

public class Base64Encoder implements Encoder {

    @Override
    public String encode(byte[] messageToEncrypt) {
        return Base64.encodeToString(messageToEncrypt, Base64.DEFAULT);
    }

    @Override
    public byte[] decode(String messageToDecrypt) {
        return Base64.decode(messageToDecrypt, Base64.DEFAULT);
    }
}

File: kfingerprintmanager/src/main/kotlin/com/jesusm/kfingerprintmanager/encryption/Base64Encoder.kt
package com.jesusm.kfingerprintmanager.encryption

import android.util.Base64

class Base64Encoder : Encoder {
    override fun encode(messageToEncode: ByteArray): String = Base64.encodeToString(messageToEncode, Base64.DEFAULT)

    override fun decode(messageToDecode: String): ByteArray = Base64.decode(messageToDecode, Base64.DEFAULT)
}